SharedCode/src/commonMain/kotlin/ApplicationPresenter.kt

```diff
@@ -72,6 +70,11 @@ class ApplicationPresenter: ApplicationContract.Presenter() {
         }
     }
 
+    override fun getTimeNow(): String {
```

Comment:
Is this function ever actually called from the view? Does it need to be on the contract?

Reply:
It's used when the advanced search time is cleared to reset the time to now